library/globals.inc.php

```diff
@@ -2211,13 +2217,13 @@
             'ms' => xl('Malay'),
             'ml' => xl('Malayalam'),
             'mt' => xl('Maltese'),
-            'mi' => xl('Māori'),
-            'mr' => xl('Marathi (Marāṭhī)'),
```

Comment:
Watch out here. Ensure your text editor is working in UTF8 mode or else you'll break characters as is done here.